### PR TITLE
Laiba add to team

### PR DESCRIPTION
**Pull request by Zamm178**

### CODE REVIEW

File: _data/team.yml [hunk 1]
```diff
@@ -354,3 +354,10 @@ people:
   pic: kevin_chan.jpg
   position: Authentication
   team: alumni
+  
+- name: Laiba Zaman
+  lastname:  Zaman
```
Comment (ljdursi) [inline _data/team.yml]: Could you remove the extra space before your last name?

File: _data/team.yml [hunk 2]
```diff
@@ -354,3 +354,10 @@ people:
   pic: kevin_chan.jpg
   position: Authentication
   team: alumni
+  
+- name: Laiba Zaman
```
Comment (ljdursi) [inline _data/team.yml]: Please move your entry up into the "implementation" section with other current team members

**Review by ljdursi — CHANGES_REQUESTED** [review]
Looks good - thanks for doing this so quickly!  I made two comments - I assume one of the two is the reason why your name is appearing out of order, and having your name up with the other current team members will make the file easier to read either way.

**Comment by Zamm178** [comment]
I've updated the file with the changes!

**Review by ljdursi — APPROVED** [review]
Looks great, thanks!  Merge it in when ever you're ready.

**Pull request merged**